### PR TITLE
Header icons

### PR DESCRIPTION
**Pull request by Meghana-12**
related to issue  #801
Changed the Icons colour to Ignitus Blue
Student: 
student
Professor: 
Prof

### CODE REVIEW

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts [hunk 1]
```diff
@@ -78,7 +78,9 @@ export const NavigationLinks = styled.ul<ToggleProps>`
 `;
 
 export const NavigationLinkItem = styled.li<NavigationProps>`
-  padding: 0rem 1rem;
+  // padding: 0rem 1rem;
```
Comment (Paarmita) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts:81]: Remove commented code

Comment (Meghana-12) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts:81]: ok

**Comment by Meghana-12** [comment]
> @Meghana-12 Can you please change the clack color of the icon to Ignitus blue.

sure

**Comment by Meghana-12** [comment]
> clack

I did, but I forgot to change the images in the PR description.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Images/img-Svg/Professor.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (divyanshu-rawat) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Images/img-Svg/Professor.svg:1]: Please put those SVGs in icons.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts [hunk 3]
```diff
@@ -78,7 +78,8 @@ export const NavigationLinks = styled.ul<ToggleProps>`
 `;
 
 export const NavigationLinkItem = styled.li<NavigationProps>`
-  padding: 0rem 1rem;
+  padding: ${props =>
+    props.padding === 'forEmail' ? '0rem 1rem 0rem 0rem' : '0rem 1rem'};
```
Comment (divyanshu-rawat) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts:82]: what is the purpose of passing `padding` props?

Comment (Meghana-12) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts:82]: oh, I forgot to remove those! My bad, I'll remove it asap.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Images/img-Svg/Professor.svg (hunk 2 above)
Comment (Meghana-12) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Images/img-Svg/Professor.svg:1]: okay

**Comment by divyanshu-rawat** [comment]
@Meghana-12 made some code improvement  feel free to have a look :)

**Pull request merged**